dartsim/src/JointFeatures.cc

```diff
@@ -466,9 +500,16 @@ Identity JointFeatures::AttachFixedJoint(
   {
     // child already has a parent joint
     // TODO(scpeters): use a WeldJointConstraint between the two bodies
-    return this->GenerateInvalidId();
+    auto worldId = this->GetWorldOfModelImpl(
```

Comment:
For one of the Physics folks: Should we snap the model to the joint position as well in this case? AddJoint automatically snaps two items together. Adding a constraint however doe not do that.